incubator/hnc/plugin/hnc.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: hnc
+spec:
+  version: "v0.0.1"
+  platforms:
+    - selector:
+        matchLabels:
+          os: linux
+          arch: amd64
+      uri: https://github.com/kubernetes-sigs/multi-tenancy/releases/v0.0.1.tar.gz
+      sha256: "5c7b88db640b12f0fb4794392e7b9574115a3814e9783966d56144449d459c49"
+      files:
+        - from: "bin/kubectl/kubectl-hnc"
+          to: "."
+      bin: "./kubectl-hnc"
+  shortDescription: namespaces hierarchical administration.
+  homepage: https://github.com/kubernetes-sigs/multi-tenancy
+  caveats: |
+    Usage:
+       kubectl hnc [command]
+
+     HNC documentation: 
+       https://github.com/kubernetes-sigs/multi-tenancy
+
+  description: |
```

Comment:
I'm not sure what this adds? Pointing people to the github repo seems enough to me.